homework9/src/components/SubscribeModal/SubscribeModal.js

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+
+import "./SubscribeModal.css";
+
+function SubscribeModal({ open, onClose }) {
+  const { register, handleSubmit } = useForm();
+
+  const navigate = useNavigate();
+
+  const onSubmit = (data) => {
+    axios
+      .post("http://localhost:5000/form/subscribe", data)
```

Comment:
for base url create .env file with variable for that and reuse it

.env
REACT_APP_API_HOST=http://localhost:5000

.SubscribeModal
.post(`${process.env.REACT_APP_API_HOST}/form/subscribe`, data)

Reply:
for API request create separate file inside `api` folder and import here